core/src/main/scala/org/http4s/CacheDirective.scala

```diff
@@ -25,51 +25,133 @@ sealed trait CacheDirective extends Product with Renderable {
+    def name = maxAge.name
     override def value: String = name.toString + "=" + deltaSeconds.toSeconds
   }
+  object maxAge {
+    val name = ci"max-age"
+  }
+  @deprecated("Renamed to maxAge", "0.22.0-M6")
+  type `max-age` = maxAge
+  @deprecated("Renamed to maxAge", "0.22.0-M6")
+  val `max-age` = maxAge
 
-  final case class `max-stale`(deltaSeconds: Option[Duration] = None) extends CacheDirective {
+  final case class maxStale(deltaSeconds: Option[Duration] = None) extends CacheDirective {
+    def name = maxStale.name
     override def value: String = name.toString + deltaSeconds.fold("")("=" + _.toSeconds)
   }
+  object maxStale {
+    val name = ci"max-stale"
+  }
+  @deprecated("Renamed to maxStale", "0.22.0-M6")
+  type `max-stale` = maxStale
+  @deprecated("Renamed to maxStale", "0.22.0-M6")
+  val `max-stale` = maxStale
 
-  final case class `min-fresh`(deltaSeconds: Duration) extends CacheDirective {
+  final case class minFresh(deltaSeconds: Duration) extends CacheDirective {
+    def name = minFresh.name
     override def value: String = name.toString + "=" + deltaSeconds.toSeconds
   }
+  object minFresh {
+    val name = ci"min-fresh"
+  }
+  @deprecated("Renamed to minFresh", "0.22.0-M6")
+  type `min-fresh` = minFresh
+  @deprecated("Renamed to minFresh", "0.22.0-M6")
+  val `min-fresh` = minFresh
 
-  case object `must-revalidate` extends CacheDirective
+  case object mustRevalidate extends CacheDirective {
+    val name = ci"must-revalidate"
+  }
+  @deprecated("Renamed to mustRevalidate", "0.22.0-M6")
+  val `must-revalidate` = mustRevalidate
 
-  final case class `no-cache`(fieldNames: List[CIString] = Nil) extends CacheDirective {
+  final case class noCache(fieldNames: List[CIString] = Nil) extends CacheDirective {
+    def name = noCache.name
     override def value: String =
       name.toString + (if (fieldNames.isEmpty) "" else fieldNames.mkString("=\"", ",", "\""))
   }
+  object noCache {
+    val name = ci"no-cache"
+  }
+  @deprecated("Renamed to noCache", "0.22.0-M6")
+  type `no-cache` = noCache
+  @deprecated("Renamed to noCache", "0.22.0-M6")
+  val `no-cache` = noCache
 
-  case object `no-store` extends CacheDirective
+  case object noStore extends CacheDirective {
+    val name = ci"no-store"
+  }
+  @deprecated("Renamed to noStore", "0.22.0-M6")
+  val `no-store` = noStore
 
-  case object `no-transform` extends CacheDirective
+  case object noTransform extends CacheDirective {
+    val name = ci"no-transform"
+  }
+  @deprecated("Renamed to noTransform", "0.22.0-M6")
+  val `no-transform` = noTransform
 
-  case object `only-if-cached` extends CacheDirective
+  case object onlyIfCached extends CacheDirective {
+    val name = ci"only-if-cached"
+  }
+  @deprecated("Renamed to onlyIfCached", "0.22.0-M6")
+  val `only-if-cached` = onlyIfCached
 
   final case class `private`(fieldNames: List[CIString] = Nil) extends CacheDirective {
```

Comment:
I left this one.  Should I have?  If not, what shall we call the ones that clash with keywords?

Reply:
Mitigated through capitalization.